index.js

```diff
@@ -144,7 +223,25 @@ function getAllMoviesReleasedAtOrBeforeYear() {}
       { "James and the Giant Peach": "91%" },
     ];
  */
-function getRottenTomatoesScoreByMovie() {}
+function getRottenTomatoesScoreByMovie(movies) {
+  if (!movies.length){
+    throw "error - there are no movies."
+  }
+
+  const rottenTomatoesScoreByMovie = movies.map((movie) => {
+    // console.log(movie.ratings[1].value)
+    // console.log(movie.ratings[1].value) // Not correct values of "Rotten Tomatoes" - this is the first index (which changes)
+  })
+  let movieRating = movies.find((rating) => {
+    if (rating.source === "RottenTomatoes"){
```

Comment:
```suggestion
    if (rating.source === "Rotten Tomatoes"){
```
if you look at the data, "Rotten Tomatoes" has a space between the words